shell/platform/android/io/flutter/view/TextureRegistry.java

```diff
@@ -55,4 +65,10 @@ interface OnFrameConsumedListener {
      */
     void onFrameConsumed();
   }
+
+  /** Listener invoked when a memory pressure warning was forward. */
+  interface OnLowMemoryListener {
```

Comment:
can this be called `OnTrimMemoryListener`?

Reply:
done.